docs/topics/Permissions.md

```diff
@@ -33,20 +33,20 @@ Below is a table of all current permissions, their integer values in hexadecimal
 | BAN_MEMBERS \*                | `0x0000000000000004` `(1 << 2)`  | Allows banning members                                                                                                                              |              |
 | ADMINISTRATOR \*              | `0x0000000000000008` `(1 << 3)`  | Allows all permissions and bypasses channel permission overwrites                                                                                   |              |
 | MANAGE_CHANNELS \*            | `0x0000000000000010` `(1 << 4)`  | Allows management and editing of channels                                                                                                           | T, V, S      |
-| MANAGE_GUILD \*               | `0x0000000000000020` `(1 << 5)`  | Allows management and editing of the guild                                                                                                          |              |
+| MANAGE_GUILD \*               | `0x0000000000000020` `(1 << 5)`  | Allows deleting invites, editing guild settings, guild welcome screen, and managing a guild's template, integrations (bots, webhooks, and app integrations), and the guild widget                             |              |
```

Comment:
```suggestion
| MANAGE_GUILD \*               | `0x0000000000000020` `(1 << 5)`  | Allows management and editing of the guild                                                               |              |
```

Reply:
since editing an management of the guild isn't terribly helpful for someone who doesn't already know what that means, I feel that something more descriptive would be more beneficial. Unless there is an issue with formatting, is there a good reason I'm missing for shortening the description?

Reply:
There could be a section describing what each extended permissions do (by that i mean the ones that give access to a lot of things, like manage server) so the table doesn't become too big

Reply:
@Suspense4615  What if it were changed to `Allows managing invites, settings, welcome screen, template, integrations (bots, webhooks, and app integrations), and the widget of a guild`? Then its about the same length as the description in the moderate members permission. It seems like getting up to three lines isn't a huge deal, and the added clarity would be a big improvement, without having to create a new section.

Reply:
Alternatively, if we used @PlavorSeol 's suggestion from above, (quoted below) and had each of his categories link to the relevant endpoint information, then that would shorten things and ensure detailed descriptions

He suggested we change it to `Allows deleting invites, editing guild settings and welcome screen and managing guild template, integrations and widget`

> Editing [guild settings](https://discord.com/developers/docs/resources/guild#modify-guild) and [welcome screen](https://discord.com/developers/docs/resources/guild#modify-guild-welcome-screen) and managing [guild template](https://discord.com/developers/docs/resources/guild-template#modify-guild-template), [integrations](https://discord.com/developers/docs/resources/guild#delete-guild-integration) and [widget](https://discord.com/developers/docs/resources/guild#modify-guild-widget) are all different things in the term of API, they use clearly _different_ endpoints, and my suggestion reflects them, assuming OP's intention is to clarify things.

Reply:
This is not taking upcoming features into account, while it fits with 3 lines *now*, Discord might add several features tied to that permission in the future. And I don't think adding a few lines once every now and then is sustainable on the long run

Reply:
@Suspense4615 Can you show me an example of what you think would work best?

Reply:
> @Suspense4615 Can you show me an example of what you think would work best?
something like [this](https://discord.com/developers/docs/topics/permissions#implicit-permissions)